### ISSUE TITLE
Kleidungsstück nicht produzierbar trotz Katalog

### ISSUE THREAD

**Issue by Wheazle** [mexyn/statev_v2_issues#2128]
**Character Name**
Aaron Reed

**Auftrittszeitpunkt (Datum / Uhrzeit)**
Jederzeit

**Beobachtetes Verhalten**
Folgende Kleidung kann nicht geschneidert werden: [Oberteile] [Lederjacken] - The Lost MC Lederjacke

**Erwartetes Verhalten**
Die Jacke ist im Katalog zu sehen, allerdings sind diese nicht zur Herstellung verfügbar

**Schritte um den Fehler nachvollziehen zu können**
-

**Monitorauflösung (nur wenn Falsche Darstellung in der UI)**
-

**Optional: Video / Bilder des Fehlers**
-

**Comment by CaleighRhodes** [comment]
Das ist eine Jacke, die nur Mitglieder des LostMC tragen können. Ggf. kannst du sie deshalb nicht produzieren. Ich glaube, mal gehört zu haben, dass nur "berechtigte" Schneidereien die produzieren dürfen.

**Comment by Wheazle** [comment]
> Das ist eine Jacke, die nur Mitglieder des LostMC tragen "dürfen". Ggf. kannst du sie deshalb nicht produzieren. Ich glaube, mal gehört zu haben, dass nur "berechtigte" Schneidereien die produzieren dürfen.

Hey Caleigh,
ich bin Mitglied des LostMC. Es geht eher darum das mehrere Schneiderein Sie nicht produzieren können. Unsere Schneiderei welche für unsere Kutten zuständig ist kann diese Lederjacke nicht produzieren. Eine andere Schneiderei ebenfalls nicht. Sie ist im Katalog sichtbar, aber nicht produzierbar.

VG

**Comment by CaleighRhodes** [comment]
Ja, das war mir schon klar, dass du das meinst. Vielleicht muss der Support die betreffende Schneiderei "freischalten" - zur Not frag mal Mike von den Helldogs oder Glinda, sie hat zum Schluss die HD-Kutten produziert.

**Comment by Enemyaa** [comment]
Die Jacken stammen aus einer Zeit, wo es noch keine Regel für custom Sachen gab. Mittlerweile ist 1 Teil pro  Biker Fraktion die Regel. Die Jacke wird nicht mehr herstellbar sein, es sei denn es ändert sich was an dieser Regelung. Das wiederum hab ich nicht zu entscheiden.
